tests/tests_suite_test.go

```diff
@@ -168,8 +168,20 @@ func cancel(url, username, password string) {
 	// log in to the account
 	login(url, username, password)
 
+	// remove any existing test-* apps
```

Comment:
That's a good point--I should have done it that way. We can easily extract it into a different `func` if need be in the future.